### PR TITLE
CMake: Fixed build from foreign source dir

### PR DESCRIPTION
**Pull request by niansa**

### CODE REVIEW

File: WickedEngine/CMakeLists.txt [hunk 1]
```diff
@@ -143,13 +143,13 @@ endif()
 
 add_library(dxcompiler SHARED IMPORTED)
 set_property(TARGET dxcompiler PROPERTY
-		IMPORTED_LOCATION ${CMAKE_SOURCE_DIR}/WickedEngine/${LIBDXCOMPILER} )
+                IMPORTED_LOCATION ${CMAKE_CURRENT_SOURCE_DIR}/WickedEngine/${LIBDXCOMPILER} )
```
Comment (portaloffreedom) [inline WickedEngine/CMakeLists.txt:146]: This line seems to be incorrect

**Comment by portaloffreedom** [comment]
What is purpose of these PR? I don't really see the point

**Comment by niansa** [comment]
Well, to be able to build wicked engine from within another projects

To allow a structure like this:

    - Git root
      - Game
      - Engine (submodule)

File: WickedEngine/CMakeLists.txt (hunk 1 above)
Comment (niansa) [reply WickedEngine/CMakeLists.txt]: Fixed!

**Pull request closed without merging**